eKreta2SDS/bin/eKreta2SDS.psm1

```diff
@@ -638,15 +645,23 @@ Function eKreta2Convert() {
                 Write-PSFMessage -Level Debug "OUT:$_"
             }
         } # End While
-    
+        
+        if ($teachers -is [System.Management.Automation.PSCustomObject]) { $teachers = @($teachers) }
+
```

Comment:
Ennek nem a 620-as sor környékén lenne logikusan a helye?
Meg érteni is csak félig értem, meg a társait is - beszéljük meg.